routes/lobby_api.js

```diff
@@ -0,0 +1,6 @@
+exports = module.exports = function(req, res) {
+    const request = require('request');
+    request(process.env.LOBBY_API_URL + '/'+req.query.resource, function (error, response, body) {
```

Comment:
`request(process.env.LOBBY_API_URL + '/' + req.query.resource, function (error, response, body) {`

Reply:
???

Reply:
I guess this is about the inconsistent use of whitespace inside the method

Reply:
Thanks!
Fixed